bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala

```diff
@@ -94,7 +94,7 @@ class PartnerView extends BorderPanel with BoundView[Proposal] {view =>
       BorderFactory.createEmptyBorder(2, 4, 2, 4))
     override def refresh(m:Option[Proposal]) {
 
-      val (reqTime, minTime) = ~m.map(_.proposalClass).map(r => (r.requestedTime.hours, r.minRequestedTime.hours))
+      val (reqTime, minTime) = m.map(_.proposalClass).map(r => (r.requestedTime.hours, r.minRequestedTime.hours)) | ((0.0, 0.0))
```

Comment:
It looks like it was using `Monoid[Double]` which shouldn't exist ... not sure how this ever worked.